syntax = "proto3";

package registration;

service Registration {
  rpc RegisterService (RegisterServiceRequest) returns (RegisterServiceResponse) {}
}

message RegisterServiceRequest {
  string name    = 1;
  string service = 2;
  string address = 3;
}

message RegisterServiceResponse {
  string message = 1;
  string status  = 2;
}
